sdk/storage/Azure.Storage.DataMovement/src/Models/SingleTransferOptions.cs

```diff
@@ -106,5 +103,13 @@ public bool Equals(SingleTransferOptions obj)
         public event SyncAsyncEventHandler<TransferFailedEventArgs> TransferFailed;
 
         internal SyncAsyncEventHandler<TransferFailedEventArgs> GetFailed() => TransferFailed;
+
+        /// <summary>
+        /// Number of single transfers skipped during Transfer due to no overwrite allowed as specified in
+        /// <see cref="CreateMode"/>
```

Comment:
nit: Maybe this comment should read more like the one above? It seems like this isn't really the number of skipped transfers but rather a collection of those skipped transfers?